cmd/loom/addressmapper.go

```diff
@@ -128,11 +135,26 @@ func ListMappingCmd() *cobra.Command {
 			if err != nil {
 				return errors.Wrap(err, "static call contract")
 			}
-			out, err := formatJSON(&resp)
-			if err != nil {
-				return errors.Wrap(err, "format JSON response")
+			type maxLength struct {
+				From int
+				To   int
+			}
+			ml := maxLength{From: 50, To: 50}
+			toList := make(map[string]bool)
+			for _, value := range resp.Mappings {
+				fromAddr := loom.UnmarshalAddressPB(value.From).String()
+				toAddr := loom.UnmarshalAddressPB(value.To).String()
+				if !toList[fromAddr] {
```

Comment:
This is kind of confusing. Address mapper is 1-1 mapping between 2 addresses, so you can just check if the address is found before. To simplify the algorithm, we can do
```
addressList := make(map[string]bool)
for _, value := range resp.Mappings {
  fromAddr := loom.UnmarshalAddressPB(value.From).String()
  toAddr := loom.UnmarshalAddressPB(value.To).String()
  if addressList[fromAddr] || addressList[toAddr] {
     continue
  }
  addressList[toAddr] = true
  addressList[fromAddr] = true
  fmt.Printf("%-*s | %-*s |\n",
      ml.From, fromAddr,
      ml.To, toAddr)
}

Reply:
I actually wanted this filtering to happen in the contract, not in the command.